src/TwigComponent/src/Resources/doc/index.rst

```diff
@@ -219,6 +219,50 @@ If an option name matches an argument name in ``mount()``, the option is
 passed as that argument and the component system will *not* try to set
 it directly on a property.
 
+ExposeInTemplate Attribute
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.1
+
+    The ``ExposeInTemplate`` attribute was added in TwigComponents 2.1.
+
+All public component properties are available directly in your component
+template. You can use the ``ExposeInTemplate`` attribute to expose
+private/protected properties and public methods as variables directly in a
+component template (`someProp` vs `this.someProp`). For properties, they
+must be *accessible* (have a getter).
+
+    // ...
+    use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+    #[AsTwigComponent('alert')]
+    class AlertComponent
+    {
+        #[ExposeInTemplate]
+        private string $message; // available as `{{ message }}` in the template
+
+        #[ExposeInTemplate('alert_type')]
+        private string $type = 'success'; // available as `{{ alert_type }}` in the template
+
+        #[ExposeInTemplate]
+        public function getIcon(): string
+        {
+            // return value will be available as `{{ icon }}` in the template (note the get prefix is trimmed)
+        }
+
+        public function getMessage(): string
```

Comment:
I liked how you had a comment above this in the PR readme, and I think it should be here too:

```
// Required to access $this->message